src/Features/Core/Portable/Diagnostics/DocumentAnalysisExecutor.cs

```diff
@@ -156,9 +156,9 @@ public async Task<IEnumerable<DiagnosticData>> ComputeDiagnosticsAsync(Diagnosti
                     _logPerformanceInfo, getTelemetryInfo: false, cancellationToken).ConfigureAwait(false);
                 return resultAndTelemetry.AnalysisResult;
             }
-            catch (OperationCanceledException)
+            catch (Exception)
```

Comment:
```suggestion
            catch
```

Reply:
Sure, a code style analyzer that fades it out would be great :-)